_docs/geyser/commands.md

```diff
@@ -36,6 +46,28 @@ Besides commands, there are other permissions that are used to guard specific Ge
 {:.table .table-striped .table-bordered}
 | Permission | Description |
 | `geyser.settings.server` | Allow players to use the [game settings menu](/img/wiki/game_menu.png). (Also requires player to have op.) |
+| `geyser.settings.gamerules` | Defines whether a user can alter gamerules in the [game settings menu](/img/wiki/game_menu.png). (Also requires player to have op.) |
+| `geyser.update` | Whether this player will receive Geyser update notifications upon joining. |
+
+## Permissions when using Geyser-Standalone/Geyser-ViaProxy
+Geyser-ViaProxy and Geyser-Standalone feature their own basic permission handler. To edit base permissions,
+open the `permissions.yml` file and add or remove permissions that Geyser players should receive when joining.
+
+You can further customize permission handling by creating a [Geyser extension](/geyser/extensions/) that makes use of Geysers API 
+to deal with permission checking.
+
+## Permissions when using Geyser on platforms without permission handlers
+The following affects Geyser-BungeeCord, Geyser-Velocity, and Geyser-Fabric. 
+These platforms unfortunately do not have built-in permission handlers. Because of that, you will need to grant permissions on those platforms 
+manually using permission handlers, such as [LuckPerms](https://luckperms.net/). 
+
+To resolve:
+- Install [LuckPerms](https://luckperms.net/) on the platform where you've installed Geyser (BungeeCord/Velocity/Fabric).
+- To allow Geyser to automatically register permission defaults, install the [LuckLink](https://github.com/onebeastchris/LuckLink) Geyser extension
+by downloading the `LuckLink.jar` and adding it to Geyser's `extensions` folder.
+- Restart the server, and permissions should be automatically registered.
+
+Alternatively, you can manually grant the necessary permissions using luckperms. See the table above for all of them.
```

Comment:
For the first paragraph, I'd suggest adding "Geyser's permission are listed above. For extension permissions, consult their documentation"

For the bullet points, I would title/preface it as an optional third-party step that will automatically register permissions for Geyser and extensions (if their extensions are using the register perm api)

I think the last sentence can be removed because it is communicated in the first paragraph.